ya-runtime-basic-auth/src/main.rs

```diff
@@ -10,23 +9,19 @@ use tokio::fs::OpenOptions;
 use tokio::process::{Child, Command};
 use ya_runtime_sdk::*;
 
-const AUTH_ERIGON_USER_PREFIX: &str = "erigolem";
-
 #[derive(Deserialize, Serialize)]
-pub struct ErigonConf {
-    erigon_http_addr: String,
-    erigon_http_port: String,
+pub struct BasicAuthConf {
+    service_prefix: String,
     public_addr: String,
     passwd_tool_path: String,
     passwd_file_path: String,
     password_default_length: usize,
 }
 
-impl Default for ErigonConf {
+impl Default for BasicAuthConf {
     fn default() -> Self {
-        ErigonConf {
-            erigon_http_addr: "127.0.0.1".to_string(),
-            erigon_http_port: "8555".to_string(),
+        BasicAuthConf {
+            service_prefix: "erigolem".to_string(),
```

Comment:
This runtime is no longer erigon-specific so service prefix shouldn't be as well. How about simply `yagna_service`?